backend/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -167,9 +163,11 @@ def __init__(self, root, url, org):
            self.urls[0] += '/'
        # Make sure root paths are created
        if not os.path.isdir(self.root):
-           fileutils.makedirs(self.root, user=APACHE_USER, group=APACHE_GROUP)
+           with cfg_component() as CFG:
+               fileutils.makedirs(self.root, user=CFG.httpd_user, group=CFG.httpd_group)
        if not os.path.isdir(self.pkgdir):
-           fileutils.makedirs(self.pkgdir, user=APACHE_USER, group=APACHE_GROUP)
+           with cfg_component() as CFG:
+               fileutils.makedirs(self.pkgdir, user=CFG.httpd_user, group=CFG.httpd_group)
```

Comment:
@sbluhm please try to avoid splitting in distinct parts, it's better to make it the following way:
```suggestion
       with cfg_component(None) as CFG:
           if not os.path.isdir(self.root):
               fileutils.makedirs(self.root, user=CFG.httpd_user, group=CFG.httpd_group)
           if not os.path.isdir(self.pkgdir):
               fileutils.makedirs(self.pkgdir, user=CFG.httpd_user, group=CFG.httpd_group)
```